src/util/fs.js

```diff
@@ -532,7 +532,11 @@ export async function copyBulk(
       const cleanup = () => delete currentlyWriting[data.dest];
       reporter.verbose(reporter.lang('verboseFileCopy', data.src, data.dest));
       return (currentlyWriting[data.dest] = readFileBuffer(data.src)
-        .then(d => {
+        .then(async (d) => {
```

Comment:
Yep, we don't seem to do anything concurrent here, so it could be refactored to use `await`